core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -795,22 +795,9 @@ class ZkUtils(val zkClient: ZkClient,
     * client updates the path stat.version gets incremented
     */
   def getSequenceId(path: String, acls: java.util.List[ACL] = DefaultAcls): Int = {
-    try {
+      makeSurePersistentPathExists(path)
```

Comment:
Although, looking at the code that was removed, it does reference `BrokerSequenceIdPath`, which seems wrong for a method called `getSequenceId`.